app/navigation/Navigator.scala

```diff
@@ -122,11 +122,24 @@ class Navigator @Inject()() {
   private def navigationForImports(userAnswers: UserAnswers, mode: Mode): Call = {
     userAnswers.get(page = ImportsPage).contains(true) match {
       case true => routes.HowManyImportsController.onPageLoad(mode)
-      case false if mode == NormalMode => routes.StartDateController.onPageLoad(mode)
-      case _ => routes.CheckYourAnswersController.onPageLoad
+      case false =>
+        if (navigateToStartDate(userAnswers)) {
+          routes.StartDateController.onPageLoad(mode)
+        } else {
+          contactDetailsOrDoNotRegister(userAnswers, mode)
+        }
+      case _ =>  routes.CheckYourAnswersController.onPageLoad
```

Comment:
Case _ is not required also the if can be put in the case i.e

```
case false if navigateToStartDate(userAnswers) => routes.StartDateController.onPageLoad(mode)
case false => contactDetailsOrDoNotRegister(userAnswers, mode)
```

Reply:
Completed